src/workloads/docs/ParallelInsert.yml

```diff
@@ -184,7 +184,6 @@ AutoRun:
   Requires:
     mongodb_setup:
     - replica
-    - replica-noflowcontrol
```

Comment:
See this comment for why this went away https://github.com/mongodb/genny/pull/431#discussion_r592692086